media/admin/com_virtuemart/controllers/zasilkovna.php

```diff
@@ -35,6 +35,8 @@
 class VirtuemartControllerZasilkovna extends VmController
 {
     const ZASILKOVNA_LIMITATIONS_REMOVED_NOTICE_DISMISSED = 'zasilkovna_limitations_removed_notice_dismissed';
+    public const FROM_POST = 'fromPost';
```

Comment:
raději než konstanty, definuj třídu ConfigSessionStorage, která dědí SessionStorage a přetěžuje její metody tak, aby ukládala do vlastního namespace s fixním id a key, takže konstanty nebudou potřeba

Reply:
musel jsem to predelat, ukazalo se, ze soucasne metody prace se session budou v J5 jinak.